catalyst/moodle-mod_hvp
# fix missing hub uuid error

**The issue**

- A few months ago there was an update to the `library` submodule that added some error messages, however, the `HUB registration error - Missing Site UUID or Hub Secret. Please check your Hub registration` message was incorrect and would show even if the hub was not enabled.
- See https://h5ptechnology.atlassian.net/browse/HFP-3883
- This was fixed in a newer version of the `library` submodule

**The fix**
- Updated the library submodule to this version. Since this is a fork, I tried to update as little as possible to avoid any regressions to only updated to the exact hash needed.
- Comparing the diff, it seems to very minor things e.g. fonts. If these do end up causing regressions we can easily roll it back and handle it another way.

**Testing**
To reproduce the bug you can do:
- Ensure the hub is not enabled / not used.
- View the admin settings any page, this just triggers h5p to query the hub info and queue the error in your session.
- View any h5p, this will then dequeue these errors all at once.

After updating the library submodule, the errors are no longer shown.
